README.md

```diff
@@ -103,6 +121,22 @@ None right now.
 - **Issues**: [#hacktoberfest](https://github.com/search?&q=user%3Aappwrite+label%3Ahacktoberfest+type:issue&state=open&type=Issues) or any issue you like to help with.
 - **Notes**: Follow [Appwrite blog](https://medium.com/appwrite-io) and [Discord community](https://appwrite.io/discord) for more information and help.
 
+#### **Hasura** (Stickers, T-shirt)
+
+- **Swag**: T-shirt, stickers
+- **Requirements**:
+  - Submit a PR with the label hacktoberfest: Swags
```

Comment:
```suggestion
  - Submit a PR with the label hacktoberfest: Swag
```

Reply:
Page doesn't mention anything specifc as of now. They have mentioned though that they will announce in future. Any suggestion?